playwright-e2e/dsm/component/navigation/navigation.ts

```diff
@@ -64,6 +64,50 @@ export class Navigation {
     await this.selectFrom(MainMenuEnum.MISCELLANEOUS, miscName);
   }
 
+  public async getDisplayedMainMenu(): Promise<MainMenuEnum[]> {
+    let menuOptions: Locator[] = [];
+    await expect(async () => {
+      menuOptions = await this.page.locator(`//app-navigation//ul[1]//li[contains(@class, 'dropdown')]//b`).all();
```

Comment:
Same one menu option could have multiple `b` element if text are in mixed cases or specially formated. So this xpath to find menu options could give wrong result.

As you said already, Menu and menu option could be generalized to be in a separate class by itself. Is existing class `dropdown.ts` not working at all for this?